apps/src/music/views/Video.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React, {
+  useRef,
+  useContext,
+  useLayoutEffect,
+  useEffect,
+  useState,
+} from 'react';
+import styles from './video.module.scss';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {AnalyticsContext} from '../context';
+
+function useWindowSize() {
+  const [size, setSize] = useState([0, 0]);
+  useLayoutEffect(() => {
+    function updateSize() {
+      const width = document.documentElement.clientWidth;
+      const height = document.documentElement.clientHeight;
+      setSize([width, height]);
+    }
+    window.addEventListener('resize', updateSize);
+    updateSize();
+    return () => window.removeEventListener('resize', updateSize);
+  }, []);
+  return size;
+}
+
+/**
+ * Renders a simple modal video player.
+ */
+const Video = ({id, onClose}) => {
```

Comment:
Can we reuse the standaloneVideo view here? Ok to not if that's too much of a refactor

Reply:
Since that received some big updates for its long-term home, it felt easier to just restore this copy.  I expect we will delete the code in this PR in the relatively near future.